platform/android/CHANGELOG.md

```diff
@@ -15,6 +15,33 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 ### Bug fixes
  - Fixed a rendering issue of `collisionBox` when `text-translate` or `icon-translate` is enabled. [#15467](https://github.com/mapbox/mapbox-gl-native/pull/15467)
 
+## 8.4.0-alpha.1 - September 4, 2019
+[Changes](https://github.com/mapbox/mapbox-gl-native/compare/android-v8.3.0...android-v8.4.0-alpha.1) since [Mapbox Maps SDK for Android v8.3.0](https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.3.0):
+
+### Features
+- Bindings for aggregated cluster properties [#15425](https://github.com/mapbox/mapbox-gl-native/pull/15425)
+- Add font array attribute configuration [#15488](https://github.com/mapbox/mapbox-gl-native/pull/15488) 
+- Expose camera padding option [#15444](https://github.com/mapbox/mapbox-gl-native/pull/15444) 
+- Add number format expression [#15424](https://github.com/mapbox/mapbox-gl-native/pull/15424) 
+
+### Bug fixes
+- Initialize CameraPositionActivity listeners after the style is loaded [#15484](https://github.com/mapbox/mapbox-gl-native/pull/15484)
```

Comment:
this is a testapp fix, no need to include this in sdk changelog

Reply:
Removed